### DISCUSSION TITLE
多次点击配置导出可能会导致登录过期

### DISCUSSION THREAD

**Discussion by tianpeidong** [apolloconfig/apollo#4110] [Q&A]
![image](https://user-images.githubusercontent.com/39491687/142570394-e9b4d7eb-c3e0-4b73-8198-db59f1804f8c.png)
多次点击这个导出按钮可能导致登录过期，跳转回登录页面，点击次数随机
当前版本：apolloconfig/apollo-portal:1.7.1
麻烦帮忙看看，谢谢

**Comment by nobodyiam** [comment]
Is there any clue to this issue, e.g. error log? 
What is the deployment structure of apollo-portal? How many instances of apollo-portal are there?
